### PR TITLE
Attempt readme improvements for new users

### PR DESCRIPTION
**Pull request by aaronsteers**
I was trying to work through this as a first time user and I wanted to share back what (I think) is expected of new users. Would someone check and provide feedback if this is correct as-is, or if I'm misunderstood anything.

Happy to make any necessary edits if this can help the next person.

### CODE REVIEW

**Comment by aaronsteers** [comment]
Attribution @smomni in slack: https://getdbt.slack.com/archives/C013MLFR7BQ/p1598850325010600?thread_ts=1598484992.010000&cid=C013MLFR7BQ

**Comment by aaronsteers** [comment]
Added this hyperlink into the relevant section of the PR: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html

File: README.md [hunk 1]
```diff
@@ -17,6 +17,24 @@
...
+To start, you will need an S3 bucket, for instance `my-staging-bucket` and an Athena database:
+
+```sql
+CREATE DATABASE IF NOT EXISTS analytics_dev
+COMMENT 'Analytics models generated by dbt (development)'
+LOCATION 's3://my-staging-bucket/'
+WITH DBPROPERTIES ('creator'='Foo Bar', 'email'='foo@bar.com');
+```
+
+Take note of your AWS region code (e.g. `us-west-2` or `eu-west-`).
```
Comment (Tomme) [inline README.md:31]: `eu-west-` - quick typo - I'm guessing it is meant to be `eu-west-1`?

File: README.md [hunk 2]
```diff
@@ -17,6 +17,24 @@
 
 `pip install git+https://github.com/Tomme/dbt-athena.git`
 
+### Prerequisites
+
+To start, you will need an S3 bucket, for instance `my-staging-bucket` and an Athena database:
```
Comment (Tomme) [inline README.md:22]: You can also create databases via AWS Glue: https://aws.amazon.com/glue/?whats-new-cards.sort-by=item.additionalFields.postDateTime&whats-new-cards.sort-order=desc Might be worth adding a footnote about that as well.

**Review by Tomme — CHANGES_REQUESTED** [review]
Thanks a lot for this, improvements to the README are always very welcome! I've added just a few comments, feel free to clarify them and I am happy to merge in your changes 👍

File: README.md (hunk 2 above)
Comment (aaronsteers) [reply README.md:22]: Added! 😄 ✅

File: README.md (hunk 1 above)
Comment (aaronsteers) [reply README.md]: Thanks for catching this. Now resolved. ✅

**Review by aaronsteers — COMMENTED** [review]
> Thanks a lot for this, improvements to the README are always very welcome! I've added just a few comments, feel free to clarify them and I am happy to merge in your changes 👍

@Tomme - Thanks for these comments. I've incorporated the suggested changes. If it looks good to you, feel free to merge. 👍

**Review by Tomme — APPROVED** [review]

**Pull request merged**